src/test/java/core/basesyntax/service/impl/FileReaderImplTest.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import core.basesyntax.service.FileReader;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Test;
+
+public class FileReaderImplTest {
+    private static final String DATA_BASE = "src/main/resources/database.csv";

```

Comment:
oh sorry, didn't notice it when I first checked, but you should use separate files for testing